### ISSUE TITLE
You should try out Gitbook

### ISSUE THREAD

**Issue by krrishd** [tooling/book-of-modern-frontend-tooling#67]
This is a really awesome project, but what I think would be really cool is if this was ported as a [GitBook](http://gitbook.io)

Essentially, that would allow you to write everything in Markdown, push via Git, and you'd have a web version that looks similar to what you have now, as well as .epub, .pdf, and .mobi compilation.

**Comment by addyosmani** [comment]
I've been considering the pros/cons of us using GitBook for this, but am not familiar with how well this might work for a fully free, open-source book. Are there other similar projects using GB?

**Comment by AaronO** [comment]
Hi @addyosmani,

GitBook co-author/co-founder here.

GitBook works great with fully free, open-source books. Here's a few open-source books written using GitBook (there's a lot more) :
* Jeremy Saenz (author of the martini framework) wrote [Building Web Apps with Go](https://github.com/codegangsta/bwag) (on [GitBook.com](http://codegangsta.gitbooks.io/building-web-apps-with-go/))
* @SamyPesse and I wrote [Learn Javascript](https://github.com/GitbookIO/javascript) (on [GitBook.com](http://gitbookio.gitbooks.io/javascript/))
* And many others on everything from [Docker](https://github.com/yeasy/docker_practice) to [Gulp](http://rookieone.gitbooks.io/learning-gulp/), robots, [Ruby for kids](http://mrdougwright.gitbooks.io/rubykin/)

Open source is part of our values: our book format and toolchain is open source as well as many other small utilities, it's important to give back to the community. I think GitHub has taught us that open-source allows us to build better software collaboratively and with a similar philosophy GitBook aims to allow people to write better books & documentation.

[GitBook.com](https://www.gitbook.com) offers services such as hosting, and automatic builds (think CI for books) (web, pdf, epubs are all built and updated whenever your original markdown is updated).

It integrates nicely with GitHub through webhooks, so you can keep your current workflow and still benefit from the hosting and automatic updates.

GitBook isn't perfect, but we've done our best to build a great product based on user feedback. (The format/toolchain can be extended with custom plugins too).

If there's anything that you want to improve we're just an email or PR/issue away :)

**Comment by sindresorhus** [comment]
@AaronO The less we have to care about how to build it the better. We just want to focus on the content. GitBook sounds like would relieve us from that pain.

Do you have any package or something for Open Source books like this one?

**Comment by AaronO** [comment]
@sindresorhus I agree, that is in fact GitBook's goal, automate and streamline the "boring parts", so authors can focus on what matters (content).

Open source books are entirely free on [GitBook.com](https://www.gitbook.com), so there's no additional cost whatsoever for you :)

We just [shipped organizations a few days ago](https://www.gitbook.com/blog/features/organizations), so the book and authors can work with that structure, exactly like you would on GitHub.

What kind of package / features were you thinking about ?

**Comment by AaronO** [comment]
@addyosmani @sindresorhus The setup steps would be more or less the following :

* Signup on [GitBook.com](https://www.gitbook.com) and create an organization
* Add the authors as members of the organization (optional)
* Create the book (as part of the organization)
* Link the book to the GitHub repo (in the book's settings)
* Adapt content to follow gitbook format
* Update links in readme/description to point to the book's landing page (or content)

Let me know if you have any questions or need help along the way.

**Comment by AaronO** [comment]
@addyosmani @sindresorhus A small update: @SamyPesse wrote a PR that migrates all the content over : https://github.com/tooling/book-of-modern-frontend-tooling/pull/76.

It was pretty straightforward, give that a try and let me know if you have any questions.

**P.S**: Sorry for all the comments.
